package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "test": "jest --collectCoverage",
     "lint": "vue-cli-service lint",
+    "test": "vue-cli-service test:unit --coverage",
```

Comment:
can you say more about this change?

Reply:
@jtauber yep does exactly the same as before except it prints a coverage report after the test run. It also brings this config into line with the config in `scaife-widgets` which was already using the `vue-cli-service test:unit` plugin.

Reply:
I can actually see even more low-hanging config fruit that will make the two setups closer to homogeneous, which I think is worth doing to totally eliminate any misleading configuration differences between the two going forward.

Reply:
Seen [this page](https://cli.vuejs.org/core-plugins/unit-jest.html) for reference.  Basically it's a plugin to simplify configuring a Vue project for Jest tests.

Reply:
Jest configurations unified.